draft-ietf-add-dnr.xml

```diff
@@ -449,6 +444,13 @@
         to check the revocation status of the certificate of an encrypted DNS
         resolver are discussed in Section 10 of <xref
         target="RFC8484"></xref>.</t>
+
+	<aside><t>PKIX validation often requires downloading data from
+	external servers (e.g., certificate revocation lists, certificate
+	transparency) which requires using DNS.  Applications SHOULD NOT
+	be allowed to use an encrypted DNS server until those PKIX
```

Comment:
Thanks @danwing.

I'm less comfortable with the second part of the text as this puts a reco on (DNS) clients + may conflict  or be redundant with already existing requirements. For example, 

* RFC6125 says the following

```
   Otherwise, if the client is an automated application not directly
   controlled by a human user, then it SHOULD terminate the
   communication attempt with a bad certificate error and log the error
   appropriately.  An automated application MAY provide a configuration
   setting that disables this behavior, but MUST enable the behavior by
   default.
```

* RFC8484 relies upon 9110 which says the following:

```
If the certificate is not valid for the target URI's origin, a user
   agent MUST either obtain confirmation from the user before proceeding
   (see [Section 3.5](https://datatracker.ietf.org/doc/html/rfc9110#section-3.5)) or terminate the connection with a bad certificate
   error.
```